applications/StructuralMechanicsApplication/tests/test_prebuckling_analysis.py

```diff
@@ -91,9 +91,9 @@ def _solve_prebuckling_problem(self,mp,NumOfNodes,iterations,echo=0):
         {
             "max_iteration"         : 1000,
             "tolerance"             : 1e-6,
-            "number_of_eigenvalues" : 2,
+            "number_of_eigenvalues" : 1,
             "echo_level"            : 0,
-            "normalize_eigenvectors": true
+            "normalize_eigenvectors": false
```

Comment:
are these changed on purpose?

Reply:
Yes, just noticed the normalization is not required